### PR TITLE
Feature/layout [IMPAC-253] [IMPAC-254]

### PR DESCRIPTION
**Pull request by xaun**
Kpi apply layout, css redesign, & 1 click edit/save.

@cesar-tonnoir please review :beers:

### CODE REVIEW

**Comment by xaun** [comment]
https://github.com/maestrano/impac/pull/312 should be merged together

File: src/components/kpi/kpi.directive.coffee [hunk 1]
```diff
@@ -88,14 +95,17 @@ angular
           $scope.kpi.layout? && $scope.kpi.layout.triggered
 
         $scope.getKpiUnit = ->
-          $scope.kpi.data? && $scope.kpi.data[$scope.kpi.element_watched].unit
+          ($scope.kpi.data? && $scope.kpi.data[$scope.kpi.element_watched].unit) || ''
 
         $scope.getKpiValue = ->
-          $scope.kpi.data? && $scope.kpi.data[$scope.kpi.element_watched].value
+          ($scope.kpi.data? && $scope.kpi.data[$scope.kpi.element_watched].unit) || ''
```
Comment (cesar-tonnoir) [inline src/components/kpi/kpi.directive.coffee]: `.value` ?

File: src/stylesheets/mixins.less [hunk 2]
```diff
@@ -152,3 +152,15 @@
 .loader-style(@color: @impac-widget-loading-spinner) {
   color: @color;
 }
+
+.column-gutter(@gutter: @grid-gutter-width) {
```
Comment (cesar-tonnoir) [inline src/stylesheets/mixins.less]: careful with that: are you sure it won't override the bootstrap class on all the maestrano's dashboard?
check if you can use @impac-padding-between-widgets instead

**Comment by cesar-tonnoir** [comment]
If that's not too complicated, I think "save" should, for each kpi, actually save the kpi only if the target has changed

**Comment by cesar-tonnoir** [comment]
the add/edit could probably be improved in terms of UI, but that will be another jira card

**Comment by cesar-tonnoir** [comment]
you should add some ellipsis to make sure the caption / emphasis is not going to overlap with the kpi's limits

**Comment by xaun** [comment]
@cesar-tonnoir see commit 01343e8. This PR should be good to merge now.

**Comment by xaun** [comment]
https://github.com/maestrano/impac-angular/pull/198

**Pull request closed without merging**